src/diet.ts

```diff
@@ -362,21 +361,29 @@ function copiers(): MenuItem<Note>[] {
 function countCopies(diet: Diet<Note>): number {
   // this only counts the copies not yet realized
   // any copies already realized will be properly counted by embezzlerCount
-  const alreadyGregarious =
-    get("beGregariousCharges") > 0 ||
-    (get("beGregariousFightsLeft") > 0 &&
-      get("beGregariousMonster") === $monster`Knob Goblin Embezzler`);
+  const currentlygregarious =
+    get("beGregariousFightsLeft") > 0 &&
+    get("beGregariousMonster") === $monster`Knob Goblin Embezzler`
+      ? 1
+      : 0;
+  const gregariousCharges = get("beGregariousCharges") ?? 0 + currentlygregarious;
 
-  const gregCounts = expectedGregs().slice(alreadyGregarious ? 1 : 0);
+  // returns an array of expected counts for number of greg copies to fight per pill use
+  // the last value is how much you expect to fight per pill
+  const gregCounts = expectedGregs();
+  const defaultGreg = gregCounts[gregCounts.length - 1];
```

Comment:
I would call this a marginalGreg or a marginalExtrovermectinGregCount or something silly like that